legal-test-fixture/legal_api/__init__.py

```diff
@@ -0,0 +1,33 @@
+import os
+from flask import Flask, jsonify
+from flask_sqlalchemy import SQLAlchemy
+from flask_cors import CORS
+from legal_api.models import db
+
+# Instantiate the database
+
+
+def create_app(script_info=None):
+    # Instantiate the app
+    app = Flask(__name__)
+
+    # Enable CORS
+    CORS(app)
+
+    # Get config
+    app.config.from_object('legal_api.config.Config')
+
+    # Set up extensions
+    db.init_app(app)
+
+    # Register blueprints
+    from legal_api.api.blueprints.fixture import fixture_blueprint
```

Comment:
this package name structure suggests this in the legal_api package, which it is not and will collide. Can this be kept in this package??

Reply:
Totally separate subdirectory -- no collisions.